app/src/main/res/values/strings.xml

```diff
@@ -39,7 +39,7 @@
     <string name="skip_short_break">Skip Short Break</string>
     <string name="start_long_break">Start Long Break</string>
     <string name="skip_long_break">Skip Long Break</string>
-    <string name="skip_break">Skip Break</string>
     <string name="currently_running_service_type_key">CURRENTLY_RUNNING_SERVICE_TYPE</string>
-    <string name="pomodoro_completion_alert_message">Congrats, Pomodoro is over.\n\nIt\'s time for a break.</string>
```

Comment:
This String has it's usages too, I wonder why you removed it.

Reply:
Ohh sorry again. Same case!